sourcegraph/about
# Update Criteo logo

Closes #5512

### Changelog
- Updates Criteos logo
- Small fix for broken NavBar spacing, smallest screensize before the mobile menu kicks in

**BEFORE:**
<img width="1106" alt="image" src="https://user-images.githubusercontent.com/59381432/176277093-bfdfa6e0-59b0-4047-93ab-a5682a2c858b.png">
**NOW**:
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/59381432/176277121-131a9cee-e40b-43ca-9565-bb54ff0f5236.png">


### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure Criteos logo is updated with no breaking changes on: `/case-studies`, `/case-studies/criteo-tackles-big-code`, `pricing`
3. Ensure NavBar does not break to 2 lines at any screen size
